modules/oadp-using-enable-api-group-versions.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/advanced-topics.adoc
+
+
+:_content-type: PROCEDURE
+[id="oadp-using-enable-api-group-versions_{context}"]
+= Working with different Kubernetes API versions in the same OADP operation
```

Comment:
What does it mean to have these happen in the same _operation_. Is this the backup? Could it be backup or restore? And _operation_ is thus a superset of the possibilities here?

Reply:
I agree - "operation" is not a clear word choice here.